src/shared/Logo/ui/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { LOGO_URL } from "../lib/constants"
+
+export interface LogoProps {
+  widthPx?: number
+  heightPx?: number
+  classNameExtended?: string
+}
+
+const Logo = ({ widthPx, heightPx, classNameExtended }: LogoProps) => {
+  const defaultSize = {
```

Comment:
Once you have defined the Dimension interface and used an appropriate prop you can default assign this object to that prop.
For example: 
```
const Logo = ({ size = { height: 50, width: 120, }, className }: LogoProps) {
...
```

Reply:
It's great! I really forgot about it. Thanks, done.